content/blog/dplyr-1-1-0-is-coming-soon/index.Rmd

```diff
@@ -295,7 +295,7 @@ Now, I'll be honest, I'm being a bit tricky here.
 The new backend for `arrange()` comes with a meaningful change in behavior - it now sorts character strings in the C locale by default, rather than in the much slower system locale (American English, for me).
 We made this change for two main reasons:
 
--   Much faster performance by default
+-   Much faster performance by default, because it can re-use data.table's radix sort algorithm
```

Comment:
Maybe better to say "inspired by data.table's radix sort algorithm" since we don't literally use it, just something heavily inspired by it and close in implementation. (IIRC, we made some changes to support vctrs types generally. @DavisVaughan would know for sure, but he's out this week). Anyway, we'll get this merged ASAP. Thanks!

Reply:
Makes sense to me, thanks for clarifying!

Reply:
Had a pass but still not quite happy with it, feel free to edit further